### PR TITLE
add crayon tool

### PR DESCRIPTION
**Pull request by load0ne**
## Description

- add crayon tool (default tool)
- add crayon tool on example

## Demo

[https://dropy-online.github.io/boardy/examples/app.html](https://dropy-online.github.io/boardy/examples/app.html)

## Related Issues

#14, #15, #16, #18

### CODE REVIEW

**Comment by always-awake** [comment]
리뷰 안받나욤?

**Comment by load0ne** [comment]
@always-awake 리뷰 받습니다~ 머지되고도 가능할걸? 일단 혼자 개발 중이니까 리뷰 남기면 다음 PR 반영함.
참고로 오픈소스화 할 계획이니 리뷰나 코멘트는 가능하면 영어로 해주세요.

**Comment by always-awake** [comment]
okay 👌

File: examples/app/index.ts [hunk 1]
```diff
@@ -105,7 +118,13 @@ toolSelectElements[0].addEventListener('input', (e: any) => {
   highlightTool(0);
 });
 
-toolSelectElements[1].addEventListener('click', (e: any) => {
+toolSelectElements[1].addEventListener('input', (e: any) => {
```
Comment (always-awake) [inline examples/app/index.ts:121]: How about setting the type of event object to `MouseEvent` rather than `any`?

**Comment by aereeeee** [comment]
nice work maaaaaan 😎

File: src/modules/Painter/tools/blackcrayon.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import {ActionType} from '@/constants';
+import blackline from './blackline';
+
+export default Tool.extend(blackline, {
+  [ActionType.MouseDown]: (
+    ctx,
+    _,
+    __,
+    unit,
+  ) => {
```
Comment (aereeeee) [inline src/modules/Painter/tools/blackcrayon.ts:11]: How about get parameters as object?

File: examples/app/index.ts (hunk 1 above)
Comment (load0ne) [reply examples/app/index.ts:121]: good point~!
i'll edit. 😄

File: src/modules/Painter/tools/blackcrayon.ts (hunk 2 above)
Comment (load0ne) [reply src/modules/Painter/tools/blackcrayon.ts:11]: i was trying to save even the cost of creating a literal object,
but when I think about it now, I don't think there will be a big difference in performance.

thanks for good feedback. i'll edit 👍

**Pull request merged**